### DISCUSSION TITLE
Migration from Razzle 3 to Razzle 4

### DISCUSSION THREAD

**Discussion by mbrochh** [jaredpalmer/razzle#1573] [Q&A] [answered]
Congrats on releasing Razzle 4!! I had a look at the tag and wow, that's a huge list of changes!

Before I try to just upgrade... is there a migration guide somewhere? Or can I just bump the version and it'll all still work? :)

**Comment by fivethreeo** [comment, marked as answer]
https://razzlejs.org/docs/upgrade-guide

**Reply by mbrochh** [reply]
Oh man. How did I not find that? Thanks a lot!

**Reply by mbrochh** [reply]
Happy to report that upgrading has worked quite well.

I got an error `Module not found: Error: Can't resolve 'crypto'` because of webpack (I think), so I had to add this to `package.json` (see https://stackoverflow.com/a/54162447):

```
  "browser": {
    "crypto": false
  }
```
